Formula/step.rb

```diff
@@ -16,8 +16,8 @@ class Step < Formula
   depends_on "go" => :build
 
   resource "certificates" do
-    url "https://github.com/smallstep/certificates/releases/download/v0.17.3/step-ca_0.17.3.tar.gz"
-    sha256 "56a7a2a1e0aec520ab62ae10da659bae329b3325c5023d6de332c6320e0f8d3e"
+    url "https://dl.step.sm/gh-release/certificates/homebrew-core/v0.17.4/step-ca_0.17.4.tar.gz"
```

Comment:
Does homebrew get different certificates from everyone else?

Reply:
Hey 👋 . 

No, this is the same certificates as everyone else. You'll notice the address for `cli` changed as well. We're trying to do a better job tracking where downloads of `cli` and `certificates` are coming from. These are just redirects that let us capture the "source" of a download. In this case, `homebrew-core`. I think it's probably meaningless here since users are downloading a bottle rather than building themselves , but I updated all the links everywhere else and wanted to be consistent. 

Our github releases page also has the updated links in the header (https://github.com/smallstep/cli/releases/tag/v0.17.5) as do our installation docs (https://smallstep.com/docs/step-ca/installation#linux).

Reply:
For our automation it's really much better if it stays on the GitHub download link and since that is still available I would like to ask you to revert the change.

Reply:
If you're interested in download statistics, you can use our [JSON API](https://formulae.brew.sh/docs/api/):
```
curl -fsS https://formulae.brew.sh/api/formula/step.json | jq '.analytics'
```

Reply:
Updated.

Reply:
Is it possible to rerun the tests? It seems to have failed trying to reach https://smallstep.com (unless I've misunderstood the test output). Momentary blip?